packages/php/project.bri

```diff
@@ -17,11 +17,15 @@ const source = Brioche.download(
 export default function php(): std.Recipe<std.Directory> {
   return std.runBash`
     ./configure --prefix="$BRIOCHE_OUTPUT"
-    make -j16
+    make -j "$(nproc)"
     make install
   `
-    .dependencies(std.toolchain, libxml2, sqlite)
     .workDir(source)
+    .dependencies(std.toolchain, libxml2, sqlite)
+    .env({
+      // Set the number of threads to use during compilation
+      OMP_NUM_THREADS: "16",
```

Comment:
Funny enough, on my computer, with 16 threads, building PHP took one more minute (9m). Whereas with 8 threads, I was able to build it in 8m.